prov/usnic/src/usdf_dgram.h

```diff
@@ -41,9 +41,8 @@
 
 #define USDF_DGRAM_CAPS (FI_MSG | FI_SOURCE | FI_SEND | FI_RECV)
 
-#define USDF_DGRAM_14_SUPP_MODE (FI_LOCAL_MR |FI_MSG_PREFIX)
+#define USDF_DGRAM_SUPP_MODE (FI_LOCAL_MR | FI_MSG_PREFIX)
 #define USDF_DGRAM_14_REQ_MODE (FI_LOCAL_MR)
```

Comment:
Because SUPP_MODE is the same for all version but only 1.4 required FI_LOCAL_MR.